packages/core-react/index.d.ts

```diff
@@ -93,14 +93,24 @@ export type SelectProps = {
 
 declare const Select: React.FC<SelectProps>;
 
-export { 
-  Button, 
+export type ImageProps = {
+  className?: string;
+  src: string;
```

Comment:
based on the source for Image, things will break if `src` or `aspect` are not supplied. That's why I have them as required.